### PR TITLE
Pipline and package update

### PR DESCRIPTION
**Pull request by AminTi**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,16 +1,94 @@
...
-# To learn more about Azure Pipelines: https://aka.ms/learn-azure-pipelines
+{
+  "manifestVersion": 1,
+  "id": "team-calendar",
+  "publisher": "ms-devlabs",
+  "version": "2.0.16",
+  "name": "Team Calendar",
+  "description": "Track events important to your team, view and manage days off, quickly see when sprints start and end, and more.",
+  "public": true,
+  "categories": ["Plan and track"],
```
Comment (madkoo) [inline azure-pipelines.yml:9]: This should be Azure boards

File: scripts/packageDev.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+var exec = require("child_process").exec;
```
Comment (madkoo) [inline scripts/packageDev.js:1]: Should there be a packageRelease.js file too ?

**Review by madkoo — COMMENTED** [review]
Check my two comments

File: package.json [hunk 3]
```diff
@@ -43,44 +43,50 @@
...
+    "build:release": "npm run compile",
     "build:dev": "npm run compile:dev && npm run postbuild",
-    "postbuild": "npm run package-extension -- --rev-version",
+    "postbuild": "npm run package:dev -- --rev-version",
+    "package:dev": "node ./scripts/packageDev",
+    "package:release": "node ./scripts/packageRelease",
     "package-extension": "tfx extension create --manifest-globs azure-devops-extension.json src/*.json",
-    "publish-extension": "tfx extension publish --manifest-globs azure-devops-extension.json src/*.json"
+    "publish-extension": "tfx extension publish --manifest-globs azure-devops-extension.json src/*.json",
+    "dev": "webpack-dev-server --hot --progress --static ./ --https --port 8888"
```
Comment (madkoo) [inline package.json:56]: Check this command , think it has older command arguments that are not supported by the latest webpack.

**Review by madkoo — COMMENTED** [review]
Check one last commend otherwise it looks good

File: package.json (hunk 3 above)
Comment (AminTi) [reply package.json]: npm run build works as expected, added few changes

**Review by madkoo — APPROVED** [review]
👍

**Pull request merged**